lib/convection/model/cloudfile.rb

```diff
@@ -14,6 +14,7 @@ module Cloudfile
       attribute :name
       attribute :region
       attribute :splay
```

Comment:
Don't need to expose this to the Cloudfile anymore right?  Especially since we are removing the sleep logic from  cloud.rb.